pan3d/dataset_builder.py

```diff
@@ -403,11 +403,14 @@ def mesh_updated(exception=None):
     # -----------------------------------------------------
 
     def import_config(self, config_file):
-        if isinstance(config_file, str) and Path(config_file).exists():
-            with open(config_file) as f:
-                config = json.load(f)
-        else:
-            config = json.loads(config_file)
+        if isinstance(config_file, dict):
+            config = config_file
+        elif isinstance(config_file, str):
+            if Path(config_file).exists():
+                with open(config_file) as f:
+                    config = json.load(f)
```

Comment:
https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text

Reply:
Changed in [b816852](https://github.com/Kitware/pan3d/pull/33/commits/b81685249babc1f58e001ca34c9f7d61bc8db8ea)